.travis.yml

```diff
@@ -51,8 +51,12 @@ matrix:
     - php: '7.1'
       env: SYMFONY=dev-master@dev
     - php: '7.1'
+      env: SONATA_ADMIN=3.*
+    - php: '7.1'
       env: SONATA_ADMIN=dev-master@dev
     - php: '7.1'
+      env: SONATA_BLOCK=3.*
```

Comment:
Link for the lazy: https://travis-ci.org/sonata-project/SonataDoctrinePhpcrAdminBundle/jobs/262050124#L467